src/providers/azure-ad.ts

```diff
@@ -32,6 +32,13 @@ export default function AzureAD<P extends Record<string, any> = AzureADProfile>(
       },
     },
     async profile(profile, tokens) {
+      const profileObject = {
+        id: profile.sub,
+        name: profile.name,
+        email: profile.email,
+        image: "",
```

Comment:
```suggestion
        image: null,
```

Reply:
Looks like that angered TS. Shouldn't `""` be fine since `!!""` returns false?

Reply:
It will potentially save it to the database as an empty string, also when used in the front-end, people will use it as `src={user.image}` which will break.

Not sure why `null` wouldn't be accepted by TS though. :thinking:

Reply:
ea5647d Fixes it.
Unfortunately I'm not too familiar with TS so I just used separate return statements.